packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -2363,7 +2363,8 @@ describe("database test", async () => {
         const descriptor2: Descriptor = {
           ...mockDescriptor,
           id: generateId(),
-          state: descriptorState.draft,
```

Comment:
To avoid breaking the previous tests, I decided to not use `draft` for those tests. I added specific `draft`-related tests below.